manifests/init.pp

```diff
@@ -404,6 +407,7 @@
   Optional[Variant[Stdlib::Absolutepath, Enum['off']]] $watchdog_device = undef,
   Enum['pcs', 'crm'] $provider                                          = 'pcs',
   String $pcs_version                                                   = '', # lint:ignore:params_empty_string_assignment
+  String[1] $highavailability_repo                                      = unfed
```

Comment:
typo `unfed`

Reply:
hi @TheMeier 
thx! 
i fixed and regenerate but somehow something is still failing and i don't find out why...
Do you have more details?